### PR TITLE
Rewrite Landing Page

### LINKED ISSUE #16: [Feature]: Rewrite Landing Page
### 🌟 Situation

The current iteration of the landing page is in a pretty bad state CSS wise and requires fixing to the point where it's faster to simply rewrite the website from scratch.

### 🎯 Task

I'd like to see the landing page rewritten with good and extensible styling source code.

### 🛠️ Action

Take the existing code base as a frame of reference but apply best practices while rewriting.

### 🏆 Result

A finished implementation of our Figma landing page design with good source code that is maintainable and extensible.

### 🙋‍♀️ Are you willing to contribute to this feature?

Yes, I'm ready to contribute!

### 📝 Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by knht**
### 🔗 Linked Issue

Closes #16 

### 📝 Description

Draft PR for harboring the landing page rewrite code

### 🛠️ Changes Made

- [ ] Change 1
- [ ] Change 2
- [ ] Change 3
TBD

### 🧪 Testing

- [ ] Unit tests added/updated
- [ ] Integration tests added/updated
- [ ] Tested manually

### ✅ Checklist

- [ ] I have read the [contributing guidelines](https://github.com/zufall-labs/.github/blob/main/CONTRIBUTING.md)
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### 👥 Reviewer Checklist

- [ ] The code change is focused and makes sense
- [ ] The code adheres to our established coding standards
- [ ] The changes are adequately tested
- [ ] The documentation is updated to reflect the changes (if necessary)

---

💖 Thank you for contributing to Zufall! 💖

### CODE REVIEW

File: app/routes/api.issues.ts [hunk 1]
```diff
@@ -57,14 +57,19 @@ export const loader: LoaderFunction = async () => {
...
 
         if (error instanceof z.ZodError) {
-            return json({ error: "Invalid environment configuration", details: error.errors }, { status: 500 });
+            return Response.json(
+                { error: "Invalid environment configuration", details: error.errors },
+                { status: 500 }
+            );
         }
 
-        return json<GitHubLoaderData>({ data: [] });
```
Comment (knht) [inline app/routes/api.issues.ts]: Why was this changed?

**Review by knht — COMMENTED** [review]
Why was the json helper from Remix removed for more boilerplate? See comments

File: app/routes/api.issues.ts (hunk 1 above)
Comment (Colin23) [reply app/routes/api.issues.ts]: Because it will be deprecated in React Routerv7.
image

This is an official flag comming from Remix. Therefore I don't think they will do any changes or handle it under the hood. See [Remix changelog](https://remix.run/docs/en/main/start/changelog) and [Remix PR](https://github.com/remix-run/remix/pull/10126). If they choose to change anything so that `json` can be used again, we could change it any time we want to. But to stay future proof and up to date with official Remix releases, I would suggest keeping my change.
